.github/workflows/release.yaml

```diff
@@ -56,6 +56,10 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
+        spark_version:
+          - 3.5.1
```

Comment:
I don't like repeating this between the two jobs, but it was the _least complex_ solution.